care/utils/assetintegration/base.py

```diff
@@ -16,6 +16,7 @@ def __init__(self, meta):
         self.host = self.meta["local_ip_address"]
         self.middleware_hostname = self.meta["middleware_hostname"]
         self.insecure_connection = self.meta.get("insecure_connection", False)
+        self.timeout = settings.MIDDLEWARE_REQUEST_TIMEOUT
```

Comment:
there is no use in setting this instance var before using, we could instead accept timeout as an optional kwarg timeout in the init method